devops/auto_node/scripts/export.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
+echo "Validator wallet private key: $(cat /.val_acc_private_key)"
+echo "Validator address: $(cat /.val_address)"
+echo "BLS private key: $(cat /.bls_private_key)"
+echo "BLS public key: $(cat /.bls_public_key)"
+echo "Network: $(cat /.network)"
+echo "Beacon Chain endpoint: $(cat /.beacon_endpoint)"
```

Comment:
why is it called beacon chain endpoint?

Reply:
Beacon chain endpoint is needed explicitly for staking transactions